src/pages/documentation/platform-assurance/tutorials/custom-validation.md

```diff
@@ -1,68 +0,0 @@
-# Custom Validation
```

Comment:
is it safe to remove this altogether or should we have a deprecation warning on top going for a while?

Reply:
Agree with Emilia - It's good practice to have a deprecation warning to provide awareness to customers.

Reply:
this assurance view was deprecated a few years ago and has been removed from the UI for well over a year. I don't think it does any good to leave this page up.

Reply:
👍 and thanks for the clarification